runner/src/main.rs

```diff
@@ -1070,7 +1070,7 @@ fn run_liche() -> Step {
                     // We exclude the following URLs from the checks:
                     // - https://groups.google.com/g/project-oak-discuss : not publicly accessible
                     // - https://crates.io/crates : returns 404 (see https://github.com/raviqqe/liche/issues/39)
```

Comment:
Could you add a line to the comment too?

Reply:
Done! Thanks :)